### ISSUE TITLE
error prone fix suggestion rejected by google-java-format

### ISSUE THREAD

**Issue by davido** [google/error-prone#881]
Running most recent EP on latest gerrit flagged this warning:

```
java/com/google/gerrit/pgm/http/jetty/ProjectQoSFilter.java:151: warning: [ClassCanBeStatic] Inner class is non-static but does not reference enclosing class
  private final class Listener implements ContinuationListener {
                ^
    (see http://errorprone.info/bugpattern/ClassCanBeStatic)
  Did you mean 'private static final class Listener implements ContinuationListener {'?
```

Running gjf (version 1.3) rejected the suggested fix:

```
diff --git a/java/com/google/gerrit/pgm/http/jetty/ProjectQoSFilter.java b/java/com/google/gerrit/pgm/http/jetty/ProjectQoSFilter.java
index 61a94bde3d..260f695a6e 100644
--- a/java/com/google/gerrit/pgm/http/jetty/ProjectQoSFilter.java
+++ b/java/com/google/gerrit/pgm/http/jetty/ProjectQoSFilter.java
@@ -148,7 +148,7 @@ public class ProjectQoSFilter implements Filter {
   @Override
   public void destroy() {}
 
-  private final static class Listener implements ContinuationListener {
+  private static final class Listener implements ContinuationListener {
     final Future<?> future;
 
     Listener(Future<?> future) {
```

**Comment by davido** [comment]
Never mind, i messed something up here.
